packages/reassure-cli/src/commands/measure.ts

```diff
@@ -22,10 +22,13 @@ export async function run(options: MeasureOptions) {
   configureLoggerOptions(options);
 
   const measurementType = options.baseline ? 'Baseline' : 'Current';
+  const date = new Date();
 
   const metadata: PerformanceMetadata = {
+    date: date.toISOString().substring(0, 10),
```

Comment:
Pls store `creationDate` as both date and time in ISO format. Do not separate it into `date` and `time` fields.